interfaces/cython/cantera/units/solution.py.in

```diff
@@ -0,0 +1,100 @@
+from .. import Solution as _Solution, PureFluid as _PureFluid, CanteraError
```

Comment:
Module does not include a single docstring. Why the `.in` suffix?

Reply:
> Why the `.in` suffix?

We use the `.in` suffix in several other places, including `setup.py.in` and `config.h.in` to name template files that are filled in by SConstruct. I believe it stands for "input", although I wouldn't swear to it. I also think that using `.in` suffix is common in the SCons documentation.